README.MD

```diff
@@ -41,11 +41,19 @@ npm config set no-proxy localhost,127.0.0.1,example.org
 Additionally proxy settings found in the environment variables `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` will be used if they are not defined in the `.npmrc` file. 
 
 ### Install a concrete revision
-If you want to specify the revision of Chromium to be installed, just set the environment variable CHROMIUM_REVISION to the number of the revision you want to install, as in:
+If you want to specify the revision of Chromium to be installed, just set the environment variable `CHROMIUM_REVISION` to the number of the revision you want to install, as in:
```

Comment:
This is a good point. I wanted to put `CHROMIUM_REVISION` as a code entry myself. Thank you ;)